### PR TITLE
initial attempt on tabbed code blocks

### PR DESCRIPTION
**Pull request by maxandersen**
basic attempt on adding tabbed blocks - but for some reason it freezes all my browsers when trying to load it :/

couldn't spot what is causing it.

### CODE REVIEW

**Comment by gsmet** [comment]
Will push some examples here once I'm done with my experiments in the main repo.

File: _guides/includes/create-app.adoc [hunk 1]
```diff
@@ -0,0 +1,60 @@
+[role=primary]
+.CLI - Maven project
```
Comment (kenyee) [inline _guides/includes/create-app.adoc:2]: This wraps weirdly on a phone FYI...

File: assets/javascript/codeBlockSwitch.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+function addBlockSwitches() {
+    $('.listingblock.primary, .sidebarblock.primary').each(function() {
+        primary = $(this);
```
Comment (Ladicek) [inline assets/javascript/codeBlockSwitch.js:3]: Nitpicking (but not really): this looks like you want to declare a local variable, but you're actually unintentionally making `primary` a global variable (this is also known as polluting the global scope). If you really want to declare a local variable (and it totally looks like that's the case here and elsewhere in this script), use `let` (or `var`, but `let` has the civilized meaning).

It's enough that all the functions declared in this script go into the global scope :-) (It seems to me that they could easily be declared inside the `$(document).ready` callback and everything would work just fine.)

Comment (gsmet) [reply assets/javascript/codeBlockSwitch.js:3]: FWIW, this is not my code :). I kept it as close as possible as upstream and fixed it to make it work with jQuery and the other things I needed.

Comment (gsmet) [reply assets/javascript/codeBlockSwitch.js:3]: @Ladicek btw, if you want to fix things, be my guest. I think we will need to tweak this thing a lot so we might as well improve it.

Comment (Ladicek) [reply assets/javascript/codeBlockSwitch.js:3]: Fair enough :-)

Comment (Ladicek) [reply assets/javascript/codeBlockSwitch.js]: Pushed a commit with this change. All functions are now declared within the on-ready callback, and all local variables are actually local variables.

File: _guides/includes/create-app.adoc (hunk 1 above)
Comment (gsmet) [reply _guides/includes/create-app.adoc]: @kenyee thanks for the heads up. @insectengine will tweak this thing to make it prettier and more compatible.

**Comment by gsmet** [comment]
I'm closing this one as James prefers a branch in the main repo to collaborate.

**Pull request closed without merging**